frontend/src/test/scala/bloop/bsp/BspProtocolSpec.scala

```diff
@@ -54,6 +54,7 @@ class BspProtocolSpec {
     test(10)
   }
 
+  @Ignore
```

Comment:
Can you open ticket about this and assign it to me?

Reply:
See #281